pkg/ctl/create/cluster.go

```diff
@@ -150,10 +150,12 @@ func newNodeGroupChecker(i int, ng *api.NodeGroup) error {
 		ng.SecurityGroups.WithShared = api.NewBoolTrue()
 	}
 
-	if ng.AllowSSH {
-		if ng.SSHPublicKeyPath == "" {
-			ng.SSHPublicKeyPath = defaultSSHPublicKey
-		}
+	if ng.AllowSSH && ng.SSHPublicKeyPath == "" {
+		ng.SSHPublicKeyPath = defaultSSHPublicKey
+	}
+
+	if ng.SSHPublicKeyPath != defaultSSHPublicKey {
```

Comment:
Actually, I think for now you just want to do this:
```suggestion
	if ng.SSHPublicKeyPath != defaultSSHPublicKey && ng.SSHPublicKeyPath != ""  {
```